typescript-server/src/app.ts

```diff
@@ -0,0 +1,46 @@
+
+/**
+ * Most type annotations in this file are not strictly necessary but are
+ * included for this example.
+ *
+ * To run this example execute the following commands to install typescript,
+ * transpile the code, and start the server:
+ *
+ */
+
+import fastify from 'fastify'
+import { createReadStream } from 'fs'
+import path from 'path';
+import { ServerResponse, IncomingMessage, Server} from 'http'
+import { AddressInfo } from 'net';
+
+const server: fastify.FastifyInstance<Server, IncomingMessage, ServerResponse> = fastify()
+
+const opts = {
+  schema: {
+    response: {
+      200: {
+        type: 'object',
+        properties: {
+          hello: {
+            type: 'string'
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
Would you mind using https://github.com/fastify/fluent-schema? It feels more natural!

Reply:
ok, I just  tried this, but met some problem , can you give some example to reference